salt/modules/artifactory.py

```diff
@@ -13,6 +13,7 @@
 import salt.utils.files
 import salt.utils.stringutils
 import salt.ext.six.moves.http_client  # pylint: disable=import-error,redefined-builtin,no-name-in-module
+from salt.utils.http import query
```

Comment:
Please use full import so that we know where the query function is coming from. See style guide:
https://docs.saltstack.com/en/latest/topics/development/conventions/style.html#imports